File: 09.js
let dias = Number(prompt('Ingrese la cantidad de días: '));
let horas = Number(prompt('Ingrese la cantidad de horas: '));
let minutos = Number(prompt('Ingrese la cantidad de minutos: '));
let segundos = Number(prompt('Ingrese la cantidad de segundos: '));

function convertirMilisegundo(d, h, m, s) {
  let ms = 0;
  ms += s * 1000;
  ms += m * 60000;
  ms += h * 3600000;
  ms += d * 86400000;
  console.log(`La cantidad de milisegundos es: ${ms}`);
}

convertirMilisegundo(dias, horas, minutos, segundos);

File: 09.py
dias = int(input("Ingresa la cantidad de dias: "))
horas = int(input("Ingresa la cantidad de horas: "))
minutos = int(input("Ingresa la cantidad de minutos: "))
segundos = int(input("Ingresa la cantidad de segundos: "))


def convertir_a_milisegundos(dias, horas, minutos, segundos):
    dias = dias * 86400000
    horas = horas * 3600000
    minutos = minutos * 60000
    segundos = segundos * 1000
    print(f'Eso da un total de: {
          dias + horas + minutos + segundos} milisegundos')


convertir_a_milisegundos(dias, horas, minutos, segundos)
